src/Microsoft.OData.Core/HttpHeaderValueLexer.cs

```diff
@@ -581,20 +562,14 @@ private sealed class HttpHeaderEnd : HttpHeaderValueLexer
             /// Constructs a new instance of <see cref="HttpHeaderEnd"/>.
             /// </summary>
             private HttpHeaderEnd()
-                : base(/*httpHeaderName*/ null, /*httpHeaderValue*/ null, /*value*/ null, /*originalText*/ null, /*startIndexOfNextItem*/ -1)
+                : base(/*httpHeaderName*/ null, /*httpHeaderValue*/ null, /*value*/ default, /*originalText*/ default, /*startIndexOfNextItem*/ -1)

```

Comment:
nit:
```suggestion
                : base(httpHeaderName: null, httpHeaderValue: null, value: default, originalText: default, startIndexOfNextItem: -1)
```

Reply:
#Resolved. I removed the argument name since it's straightforward.